extensions/cornerstone-dicom-sr/src/components/OHIFCornerstoneSRMeasurementViewport.tsx

```diff
@@ -93,22 +93,12 @@ function OHIFCornerstoneSRMeasurementViewport(props: withAppTypes) {
 
   const updateViewport = useCallback(
     newMeasurementSelected => {
-      const { StudyInstanceUID, displaySetInstanceUID, sopClassUids } = srDisplaySet;
+      const { StudyInstanceUID, displaySetInstanceUID } = srDisplaySet;
 
       if (!StudyInstanceUID || !displaySetInstanceUID) {
         return;
       }
 
-      if (sopClassUids && sopClassUids.length > 1) {
```

Comment:
Do we want to remove the warnings and comments here?

Reply:
Yes @wayfarer3130